src/main/resources/view/JobListCard.fxml

```diff
@@ -12,7 +12,7 @@
       <Label fx:id="idx" minWidth="17.0" prefHeight="17.0" prefWidth="17.0" text="1" />
       <Label fx:id="desc" layoutX="35.0" layoutY="27.0" maxWidth="100.0" minWidth="100.0" prefWidth="100.0" text="Fix washing machine" wrapText="true" />
       <Label fx:id="duration" layoutX="265.0" layoutY="27.0" maxWidth="60.0" minWidth="60.0" prefWidth="60.0" text="PT48H" wrapText="true" />
-      <StackPane alignment="CENTER_LEFT" maxWidth="80.0" minWidth="80.0" prefWidth="80.0">
+      <StackPane alignment="CENTER_LEFT" maxWidth="30.0" minWidth="30.0" prefWidth="30.0">
```

Comment:
Perhaps something to consider in future revisions might be to make use of XSLT?
Something like 
`<xsl:param name='count'>28</xsl:param>` 
`<param name='namelist' split=',' max-len='100' max-occurrences='{$count}'/>`
(taken from [source](https://stackoverflow.com/questions/49993748/declaring-and-using-a-variable-in-xml))
might make edits more convenient in general, though the files for this project are currently small enough that this probably does not add much value at the moment.

Reply:
I wish I knew about this 😭

Reply:
But i'm not sure if SceneBuilder will overwrite it when I make changes in that interface, so i'll invest my time in fixing other bugs. Thanks for the good suggestion.